usaspending_api/api_contracts/contracts/v2/references/total_budgetary_resources.md

```diff
@@ -11,35 +11,38 @@ This endpoint returns federal budgetary resources by fiscal year and fiscal peri
 
 + Parameters
 
-    + `fiscal_year`(optional, number)
+    + `fiscal_year` (optional, number)
         The fiscal year.
     + `fiscal_period` (optional, number)
-        The fiscal period. If this optional parameter is provided then `fiscal_year` is a required parameter
+        The fiscal period. If this optional parameter is provided then `fiscal_year` is a required parameter. If `fiscal_period` is provided without `fiscal_year`, a 400 error is returned.  Valid values: 1-12 (1 = October ... 12 = September). For retrieving quarterly data, provide the period which equals 'quarter * 3' (e.g. Q2 = P6). If neither paramater is provided, the entire available history will be returned.
 
 + Response 200 (application/json)
 
     + Attributes (object)
         + `results` (required, array[FederalBudgetaryResources], fixed-type)
+        + `messages` (required, array[string], fixed-type)
+            An array of warnings or instructional directives to aid consumers of this endpoint with development and debugging.
     + Body
 
             {
                 "results": [
                     {
-                        "total_budgetary_resources": 8361447130497.72,
                         "fiscal_year": 2020,
                         "fiscal_period": 6
+                        "total_budgetary_resources": 8361447130497.72,
```

Comment:
Are these actual numbers from testing the new endpoint?

Reply:
Probably not